.github/workflows/daka.yaml

```diff
@@ -2,7 +2,7 @@ name: Daka
 
 on:
   schedule:
-    - cron: '0 0,11 * * *'
+    - cron: "0 0,11 * * *"
```

Comment:
i think we need to add formatter to prevent this

Reply:
already merge prettier pr